src/core/ext/transport/chttp2/transport/flow_control.cc

```diff
@@ -296,9 +296,12 @@ FlowControlAction TransportFlowControl::PeriodicUpdate() {
     // target might change based on how much memory pressure we are under
     // TODO(ncteisen): experiment with setting target to be huge under low
     // memory pressure.
-    double target = kSmoothMemoryPressure
-                        ? TargetInitialWindowSizeBasedOnMemoryPressureAndBdp()
-                        : pow(2, SmoothLogBdp(TargetLogBdp()));
+    uint32_t target = static_cast<uint32_t>(RoundUpToPowerOf2(
+        Clamp(kSmoothMemoryPressure
```

Comment:
does anyone use `grpc_experimental_smooth_memory_presure`? If not, then, this should already be a power of 2 right?

Reply:
pow before was on floats... we do a smooth interpolation in number of bits and then turn that into a size, so we'll say 19.32 bits of initial window ==> 654484 bytes.

I'm expecting to flag flip to `grpc_experimental_smooth_memory_presure` sometime real soon though - it seems to perform really well with experiments so far.